.changes/next-release/feature-AWSSDKforJavav2-b405876.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "Amazon DyanmoDB",
```

Comment:
Even though we have additional caching for DDB, do we need a separate entry for "AWS SDK for Java v2" since the change is global?

Reply:
Added separate changelog entry